examples/sensors/imu.py

```diff
@@ -0,0 +1,139 @@
+import argparse
+
+import numpy as np
+from custom_data_handlers import IS_MATPLOTLIB_AVAILABLE, IS_PYQTGRAPH_AVAILABLE, MPLPlotter, PyQtGraphPlotter
+from tqdm import tqdm
+
+import genesis as gs
+from genesis.sensors.data_handlers import NPZFileWriter
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-v", "--vis", action="store_true", default=True)
+    parser.add_argument("-c", "--cpu", action="store_true", default=False)
+    args = parser.parse_args()
+
+    ########################## init ##########################
+    gs.init(backend=gs.cpu if args.cpu else gs.gpu)
+
+    ########################## create a scene ##########################
+    scene = gs.Scene(
+        viewer_options=gs.options.ViewerOptions(
+            camera_pos=(3.5, 0.0, 2.5),
+            camera_lookat=(0.0, 0.0, 0.5),
+            camera_fov=40,
+        ),
+        sim_options=gs.options.SimOptions(
+            dt=0.01,
+        ),
+        profiling_options=gs.options.ProfilingOptions(
+            show_FPS=False,
+        ),
+        show_viewer=args.vis,
+    )
+
+    ########################## entities ##########################
+    scene.add_entity(gs.morphs.Plane())
+    franka = scene.add_entity(
+        gs.morphs.MJCF(file="xml/franka_emika_panda/panda.xml"),
+    )
+    end_effector = franka.get_link("hand")
+    motors_dof = np.arange(7)
```

Comment:
I would prefer an explicit list for simplicity: `motors_dof = (0, 1, 2, 3, 4, 5, 6)` but as you wish.